rinja_parser/src/lib.rs

```diff
@@ -824,6 +824,15 @@ impl<'a> SyntaxBuilder<'a> {
                     "delimiters may not contain white spaces. \
                         The {k} delimiter ({s:?}) contains white spaces",
                 ));
+            } else if is_closing
+                && ['(', ')', '-', '+', '~', '.', '>', '<', '&', '|', '!']
```

Comment:
Actually, `)` should be fine. We only ever look for `)` in conjunction with a `(`. This way, if someone wants to use e.g. `(( expr ))` it would still work fine. Sorry that I didn't notice that during my first review!

Reply:
Nice catch!